var/spack/repos/builtin/packages/py-gitpython/package.py

```diff
@@ -7,14 +7,22 @@
 
 
 class PyGitpython(PythonPackage):
-    """
-    GitPython is a python library used to interact with Git repositories.
-    """
+    """GitPython is a python library used to interact with Git repositories."""
 
     homepage = "http://gitpython.readthedocs.org"
     url      = "https://pypi.io/packages/source/g/gitpython/GitPython-3.1.7.tar.gz"
 
-    version('3.1.7', sha256='2db287d71a284e22e5c2846042d0602465c7434d910406990d5b74df4afb0858')
+    version('3.1.8', sha256='49d4f859bda7bd6ef9ce72d9a2cddb6a59dd70f34a18ce409670bc655431c517')
+    version('3.1.7', sha256='974ded8ca74441cadc8d5c32123595347a7e43d6fefde439735601e6d3a8716b')
```

Comment:
Checksum for 3.1.7 changed, huh?

Reply:
@rspavel I think you added new checksums, right? Can you comment on this question.

Reply:
I don't know if I did the gitpython spackage but it looks like that may have been a transient issue since a ```spack checksum py-gitpython@3.1.7``` is back to the 97.

@rvinaybharadwaj I would say to delete that line